test/index.test.js

```diff
@@ -3,7 +3,7 @@ var fs = require("fs");
 var should = require("should");
 var loader = require("../");
 
-function execLoader(filename, callback) {
+function execLoader(filename, callback, includeModulePaths) {
```

Comment:
I am not so sure about the flag. It would feel better to have another function (`exec???Loader`).